uint128/uint128_test.go

```diff
@@ -0,0 +1,371 @@
+package uint128
+
+import (
+	"bytes"
+	"encoding/json"
+	"math/big"
+	"testing"
+
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/consensys/gnark-crypto/ecc/stark-curve/fp"
+)
+
+func TestUint128Bytes(t *testing.T) {
+	tests := []struct {
+		description    string
+		loBits         uint64
+		hiBits         uint64
+		expectedBytes  []byte
+		expectedString string
+	}{
+		{
+			description:    "128-bit #1",
```

Comment:
We can make the descriptions a little more expressive in some cases. For example, this one can be `"zero"`. The third one could be `"max"`, etc.